addons/sale_timesheet/tests/test_sale_timesheet_portal.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
Specifying the encoding is not necessary. I would however add the note about the license